src/lib/app.ts

```diff
@@ -126,9 +126,10 @@ export default function getApp(
         res.send(indexHTML);
     });
 
-    app.get('*', (req, res) => {
+    app.get(`${baseUriPath}*`, (req, res) => {
         if (req.path.includes('api')) {
             res.status(404).send();
+            return;
```

Comment:
we need to return here to avoid the next `res.send` to be triggered as well. Which is not allowed sense response headers are already sent.

Reply:
Makes sense.